tests/compute/test_pickle.py

```diff
@@ -60,6 +60,8 @@ def test_pickling_frame():
 
 
 def _assert_is_identical(g, g2):
+    assert g.is_multigraph == g2.is_multigraph
+    assert g._graph.is_readonly() == g2._graph.is_readonly()    # TODO: is_readonly interface
```

Comment:
TODO?

Reply:
I'm thinking of a `g.is_readonly` interface.